mdtraj/geometry/rdf.py

```diff
@@ -93,9 +93,9 @@ def compute_rdf(traj, pairs, r_range=None, bin_width=0.005, n_bins=None,
     return r, g_r
 
 
-def compute_rdf_t(traj, pairs, times, period_length=None, r_range=None,
-                  bin_width=0.005, n_bins=None, self_correlation=True,
-                  periodic=True, opt=True):
+def compute_rdf_t(traj, pairs, times, n_concurrent_pairs = 100000, 
```

Comment:
Small request. Can you move the new argument to the end of the argument list, or maybe right before `opt`? I think it would be preferable to have the options that control the meaning of what's computed come first, and the options that control the performance/memory/speed but not the computed values come at the end.